### ISSUE TITLE
Forced to use clKernel 0 command to get 6800xt and Rx570 8Gb to Mine

### ISSUE THREAD

**Issue by johnnyde94** [PhoenixMinerDevTeam/PhoenixMiner#48]
Hello, I seem to be running into an issue where I can not get the miner to run without using the -clkernel 0 command ether when mining on ether of my 6800xt or rx570 8Gb together or by themselves.  If I try to run kernel 1,2,3 it will give me a -47 error I believe and if I don't have any clKernel argument the miner will also give me the -47 error. I am running on up-to-date Windows 10.

**Comment by ben1628** [comment]
I'm using RX 6800 and I don't seem to have that problem.

Core 1200, voltage 800, and VRAM at 2140.  Hashrate is around 64 MH/s

**Comment by ooOOJavaOOoo** [comment]
I also must put in -clKernal 0 using the RX570

**Comment by RGlabs84** [comment]
What drivers are you using? if its the newest they aren't supported and are only supported through the command argument that you are already using

**Comment by johnnyde94** [comment]
I am using the latest WHQL driver. 

John DeLuca

**Comment by EdisAganovic** [comment]
Use drivers 20.4.2 and It will work perfectly.

**Comment by johnnyde94** [comment]
Wouldn’t that driver not have comparability with big Navi?

John DeLuca

**Comment by Hotdog-Vendor** [comment]
I think you can use upto 21.1.1 with out issues I am yet to test this thou, Currently I use clKernel 0 get 64mh/s average on 6800xt, play games on PC too hence why I havent tried yet.

**Comment by RGlabs84** [comment]
Correct. Driver 21.1.1 is current max driver or you have to use kernel 0.
However 0 kernel results in many
More stale shares

**Comment by johnnyde94** [comment]
That’s about what im getting too 63-65.

John DeLuca

**Comment by Hotdog-Vendor** [comment]
So I changed to 21.1.1, hash is now stable 64mh/s not bouncing from 63.9 to 65.4mh/s. Gets almost no stale shares now, tbh i think most of them phoenix made up as it would say 20% when it was 5%. Pool speed seems to be 61 to 66mh/s. Whole pc is very laggy when mining still thou :(

**Comment by RGlabs84** [comment]
Yes. That happens when your using the display that your mining with. Other
then that. Issue should be marked resolved until new release with newer
driver support
